magstock/templates/regextra.html

```diff
@@ -0,0 +1,116 @@
+
```

Comment:
This is where we make changes to our core reg form.  Here's a basic overview of everything which is happening here:

1) We add new form fields for all of the new database columns we defined.  We use Javascript to insert them into specific places in the form.

2) We insert ``Buy a Shirt" into the badge menu at the top of the page, and we change the wording on the "Regular Attendee" and "Supporter Pack" badge types.

3) Because there are no kick-in levels which don't already exist in the badge selection row at the top of the page, we hide the kick-in dropdown row for the main prereg form (we still show it for all of the other forms, since those other forms don't have the badge selection row).

4) We implement the usual sort of customized Javascript for the form fields, e.g. having the "allergies" row only show up if they're buying food, etc.